numba/cext/listobject.c

```diff
@@ -623,7 +623,7 @@ numba_test_list(void) {
     Py_ssize_t it_count;
     const char *it_item = NULL;
     NB_ListIter iter;
-    char got_item[4] = "\x00\x00\x00\x00";
+    char got_item[4] = {'\x00', '\x00', '\x00', '\x00'};
```

Comment:
This issue arose because C++ 11 complains about a size mismatch between the string and the array. Since the string had a null terminator I believe its length was actually 5.